packages/generator-single-spa/src/angular/generator-single-spa-angular.js

```diff
@@ -0,0 +1,8 @@
+const Generator = require('yeoman-generator')
+const { spawn } = require('child_process')
+
+module.exports = class SingleSpaAngularGenerator extends Generator {
+  async runVueCli() {
```

Comment:
nitpick: should this not be runAngularCli?

Reply:
👍 updated